collections/forms/i18n/en.pot

```diff
@@ -98,4 +95,12 @@ msgid "Please provide a string"
 msgstr "Please provide a string"
 
 msgid "Please provide a valid url"
-msgstr "Please provide a valid url"
\ No newline at end of file
+msgstr "Please provide a valid url"
+
+msgctxt " - or @"
+msgid ""
+"Please provide a username between 4 and 255 characters long and possibly "
+"separated by . "
+msgstr ""
+"Please provide a username between 4 and 255 characters long and possibly "
+"separated by . _ - or @"
```

Comment:
I think it's because that I built from source, and the translation were generated. Probably hasn't been committed since that string was added.

Reply:
Weird how we have two slightly different strings. We should only need the second message. Would be good to work out where the first is coming from and removing it there.
_I know this is unrelated, but it should not take long to do_

Reply:
I keep getting the same result when deleting and re-running build. 
Seems to be due to the "_" in the string, and since that is the same as the namespace prefix (I think), it uses `msgctxt " - or @"` to resolve it. That's why it's different.

Reply:
OK, let's leave it in then. Thanks for looking into it.